integrations/azure_devops.rst

```diff
@@ -0,0 +1,142 @@
+.. _azure_devops:
+
+
+|azure_logo| Azure DevOps
+=========================
+
+Thanks to the `JFrog Artifactory Extension for Azure DevOps and TFS`_ it is possible to support
+Conan tasks and integrate it with the CI development platform provided by `Microsoft’s Azure DevOps`_
+and the `Artifactory binary repository manager`_.
+
+The support for Conan now in the JFrog Artifactory Extension helps you perform the following
+tasks in Azure DevOps or TFS:
+
+ * Run Conan commands
+ * Resolve Conan dependencies from remote Artifactory servers
+ * Push Conan packages to Artifactory
+ * Publish BuildInfo metadata
+ * Import a Conan configuration
+
+In this section we will show you how to add Conan tasks to your pipelines using the Artifactory
+Extension and push the generated buildinfo metadata to Artifactory where it can be used to track
+and automate your builds.
+
+Configuring DevOps Azure to use Artifactory with Conan
+------------------------------------------------------
+
+To use the Conan support provided by the JFrog Artifactory Extension you must
+`configure a self-hosted agent`_
+that will enable Conan builds for your Azure Pipelines environment. Afterwards you can install
+the JFrog Artifactory Extension from the Visual Studio Marketplace and follow the installation
```

Comment:
link to the Extension would be helpful
are you talking about this one? https://marketplace.visualstudio.com/items?itemName=JFrog.JFrog
online search doesn't show others by keywords `JFrog` or `Artifactory`, but description talks about `XRay`
in Visual Studio itself, I get `No items found`.

Reply:
The link is in the first paragraph, first line, of the page, under the _"JFrog Artifactory Extension for Azure DevOps and TFS"_